src/backend/backend.h

```diff
@@ -363,10 +347,11 @@ namespace snmalloc
     }
 
     static void dealloc_chunk(
-      LocalState& local_state, SlabMetadata& meta_common, size_t size)
+      LocalState& local_state,
+      SlabMetadata& meta_common,
+      capptr::Alloc<void> alloc,
```

Comment:
Now that these are separated, should we assert that the `MetaEntry` for this `alloc` is indeed pointed at this `meta`-data field?

Reply:
Good idea.  Done.